packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -133,14 +132,6 @@ export async function getCommonConfig(wco: WebpackConfigOptions): Promise<Config
     }
   }
 
-  if (profilingEnabled) {
-    extraPlugins.push(
-      new debug.ProfilingPlugin({
```

Comment:
Nit: I'm guessing there's no way to drop a dependency on this `ProfilingPlugin` at the NPM layer? Does it come directly from Webpack in a way we can't separate it?

Reply:
That is correct.